packages/web/src/app/Layout.tsx

```diff
@@ -6,9 +6,9 @@ import Footer from './Footer';
 export function Layout() {
   return (
     <Suspense fallback="loading">
-      <div className="min-h-screen flex flex-col">
+      <div className="h-screen flex flex-col">
         <Header />
-        <div className="flex-grow relative flex flex-col max-w-[1800px] mx-auto w-full">
+        <div className="flex-grow relative flex flex-col min-h-0 max-w-[1800px] mx-auto w-full">
```

Comment:
Is there a way we can build this without changing these two classes? That will prevent us from adding scroll to this element later.

Reply:
Let me know if you know how. This is all I could come up with to make the sidebar scroll correctly.

Do we really need this element to scroll? We can just add scroll to the child element. It's much easier to control that way.

Reply:
I just pushed up a commit to fix this. The scroll issue is with the entire page. If we don't let the wrapper around the header, main content, and footer grow more than the screen height, then we can't scroll the entire page. We want the wrapper div to have a minimum height of the screen size so that the footer is always at the bottom. We can use `absolute h-full w-full` inside the main content div to get a page that is only as tall as the screen with no scrolling.